src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -52,6 +52,14 @@
       <HintPath>..\..\packages\Edge.js.4.0.0\lib\EdgeJs.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="FSharp.Core, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Where are these two F# references coming from? They used to come from us importing the old WebHooks extension, but we no longer do so I don't see why they're here?